docs/devsite-help/troubleshooting.md

```diff
@@ -1,5 +1,8 @@
 # Troubleshooting
 
+See the [support page](support.md) for suggestions about how to ask for more help
+if the sections below don't cover your question.
+
 ## How can I trace gRPC issues?
 
 When working with libraries that use gRPC, you need to be aware of which gRPC
```

Comment:
This is again confusing, is this about gRPC transport or gRPC implementation (of which the REST adapter is one example)?

For instance we will get the Client-level Logging for REGAPICs, but neither of the gRPC implementation logic.

I think if we make the distinction between protobuf libraries and discovery libraries on the Types Of Library pages and then we say the protobuf libraries may or will support REST, and we only use gRPC to refer to transport, things might be clearer.

I'm fine if that's done later though.

Reply:
Let's at least wait until later; I'm personally happier to keep it as-is and leave REGAPIC as a gRPC implementation detail, effectively.